scripts/create_xcframeworks.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+#
+# Shebang to explicitly use bash and not break on macOS boxes with zsh as default shell.
+#
+# After that you usually describe what this script is for and how to use it. In particular,
+# what it expects the current directory to be. (That's more important if it's it "scripts".)
```

Comment:
This was more of a placeholder text to be replaced rather than a literal suggestion 😄

Reply:
Oh 😄